### PR TITLE
added clarification note about analogRead

### PR DESCRIPTION
**Pull request by abcd-ca**
about continuous callback calls vs one single call

### CODE REVIEW

**Comment by fivdi** [comment]
@abcd-ca Thanks for the PR :)

I not sure if this is the correct location in the readme for this comment. Continuous reading isn't specific to `analogRead`. It's also used with `digitalRead` and `i2cRead` (both of which also involve Linux-IO).

Would it not be better to add a more general comment to section [Johnny-Five Features Supported](https://github.com/julianduque/beaglebone-io#johnny-five-features-supported) which linked to the [IO Plugins readme](https://github.com/rwaldron/io-plugins)?

**Comment by abcd-ca** [comment]
@fivdi I updated the note to also include `digitalRead`, good idea. In my opinion it belongs here for the following reasons: I was looking for a way to do analog reads and came across this plugin repo first. From there I discovered Johnny-Five. All I needed was analog read capability and not everything Johnny-Five has to offer. This plugin repo shows a standalone example ("Standalone Usage of BeagleBone-IO") that isn't dependent on Johnny-Five. My note adds clarification to that section and it would have saved me a couple hours to have seen it there. A similar note quite likely belongs on the Johnny-Five side of things too but I'm not using that so I haven't looked into it too much.

**Comment by abcd-ca** [comment]
That being said, is there a simpler alternative to doing analog reads? It was absent from the [onoff](https://github.com/fivdi/onoff) library which I am also using. Furthermore, I think for my own uses I'll end up hacking the source of the linux-io lib to not do a setInterval loop on the analog read because I really don't need a continuous read of my (battery-voltage) analog pin

File: README.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ board.on('ready', function () {
 });
 ```
 
+Note, calling `analogRead`, `digitalRead` or `i2cRead` initiates a new **continuous** data reading process for a pin not a single read. Further details can be seen at https://github.com/rwaldron/io-plugins#analogreadpin-handler.
```
Comment (fivdi) [inline README.md]: "Note," isn't needed and should be removed.
The strong emphasis on **continuous** isn't needed and should be removed.
The link https://github.com/rwaldron/io-plugins#analogreadpin-handler only applies to `analogRead`, not to `digitalRead` or `i2cRead`. It should be changed to [IO Plugins](https://github.com/rwaldron/io-plugins).

**Comment by fivdi** [comment]
> In my opinion it belongs here for the following reasons...

The reasoning here makes sense to me.

> That being said, is there a simpler alternative to doing analog reads

Yes, it's actually very simple to read a value of an analog channel. It's only necessary to read a file. The details can be seen at https://github.com/julianduque/beaglebone-io/blob/master/lib/analog-input.js.

**Comment by fivdi** [comment]
@abcd-ca Thanks.

**Pull request merged**